poly/src/evaluations.rs

```diff
@@ -146,8 +146,7 @@ impl<'a, F: FftField, D: EvaluationDomain<F>> DivAssign<&'a Evaluations<F, D>>
     #[inline]
     fn div_assign(&mut self, other: &'a Evaluations<F, D>) {
         assert_eq!(self.domain, other.domain, "domains are unequal");
-        self.evals
-            .iter_mut()
+        ark_std::cfg_iter_mut!(self.evals)
```

Comment:
I guess we should use batch inversion here? (I know the old code didn’t)

Reply:
Sure, I'll add a parallel batch inversion method.